test-cases/batch-tests/batch-tests.sh

```diff
@@ -28,5 +28,26 @@ latexindent.pl -s cmh.tex lines*.tex -g fatal-file-not-exist.log
 latexindent.pl -s cmh1.tex lines*.tex -g fatal-file-no-read.log
 
 latexindent.pl -l documentation.yaml -rv -w -m -s -g not-working.log doc*.tex
+
+# local settings found with default name (friend #1)
+latexindent.pl -s -l local-settings-friend-1/mwe.tex
+(cd local-settings-friend-1 && (cat indent.log | grep "defaultIndent" | sed 's/^ *//g' > localSettings.yaml))
```

Comment:
I don't think we need this line. Similarly for the analogous ones below.

Reply:
It's line 34 that isn't needed. Line 33 looks fine.

Reply:
Are you sure? If I understand the test concept correctly, a test fails if something tracked by git is changed, right? Line 33 will run successfully no matter if the local settings file is found or not; the command also does not modify the file or writes out the result into another file, so line 34 is required to modify the settings if they are not read correctly ("\t\t" gets replaced by "\t" if the local settings are not considered) ... so I think it is required. However, there are at least two alternatives:

1. Use the flag `-o=+out` and provide corresponding files with the desired output. 

2. Only grep the lines matching "WARN" and provide a corresponding file with the expected warnings (as I have done for the last test (lines 48-50).

I am happy to change something or remove tests if you want; they are on separate commits and could be reverted. However, I like the second alternative the most as it's more obvious what is tested.

Reply:
Yes, you've understood the test cases :) I like your option 2,that makes sense. Feel free to keep adding commits, they will ultimately be squashed into 1 when merged